packages/react-i18n/src/i18n.ts

```diff
@@ -49,7 +49,7 @@ export interface NumberFormatOptions extends Intl.NumberFormatOptions {
 }
 
 export interface CurrencyFormatOptions extends NumberFormatOptions {
-  form?: 'short' | 'explicit';
+  form?: 'auto' | 'short' | 'explicit';
```

Comment:
since this is effectively automatically selecting based on locale, should we use `locale` instead of `auto`?

Reply:
I prefer `auto`.